src/main/java/demon/Deadline.java

```diff
@@ -0,0 +1,27 @@
+package demon;
+
+import java.time.LocalDateTime;
+import java.time.format.DateTimeFormatter;
+
+/**
+ * Represents a task that has a deadline.
+ */
+public class Deadline extends Task {
+
+    protected LocalDateTime dateTime;
```

Comment:
Hmm, perhaps you can consider changing the accessibility to 'private' and use a getter? 🤔